Resources/public/js/views/universaldiscovery/ez-universaldiscoverymethodbaseview.js

```diff
@@ -97,6 +97,16 @@ YUI.add('ez-universaldiscoverymethodbaseview', function (Y) {
             },
 
             /**
+             * The location that the UDW tree will select on start
```

Comment:
you are modifying the method base view and depending on the implementation, there could be no tree, so this description is wrong in this context. You should write something like:
```
The Location Id where the content discovery can start.
```